public/assets/js/spongedownloads.js

```diff
@@ -0,0 +1,388 @@
+const API = "https://dev.minecrell.net/downloads"
```

Comment:
we already have gulp stuff for scss, we just need to make sure that the build setup won't break in the process